document how to set multitenancy in compose for 8.8

For 8.7 and below, we have a setting `MULTI_TENANCY_ENABLED` which will enable or disable multitenancy. In https://github.com/camunda/camunda-distributions/pull/185
I removed this setting because it only modified one setting, and we want to encourage people editing the `.core/application.yaml`. Additionally, the setting was incorrect so multitenancy was not working when set.

This issue is for tracking the documentation update of how we tell people how to configure multitenancy, as we previously directed users to set `MULTI_TENANCY_ENABLED` (In a branch I removed these references).

I need to test with multi-tenancy but can't get it running neither with `alpha7` nor `alpha8`:

1. With `alpha8` there is no manual how to enable mutli-tenancy (what this issue is about)
2. With `alpha7` on setting properties in `.env` as for former versions I also had to add `CAMUNDA_SECURITY_AUTHENTICATION_UNPROTECTEDAPI=false` for Zeebe but still get errors in the connectors runtime and in my application

For connectors:
```
2025-09-22 09:10:09.123 |
2025-09-22 09:10:09.152 | 2025-09-22T07:10:09.152Z ERROR 1 --- [ult-executor-60] i.c.c.i.oauth.OAuthCredentialsProvider   : Failed while fetching credentials:
2025-09-22 09:10:09.152 |
2025-09-22 09:10:09.152 | java.net.BindException: Cannot assign requested address
2025-09-22 09:10:09.152 | 	at java.base/sun.nio.ch.Net.connect0(Native Method)
2025-09-22 09:10:09.152 | 	at java.base/sun.nio.ch.Net.connect(Unknown Source)
2025-09-22 09:10:09.152 | 	at java.base/sun.nio.ch.Net.connect(Unknown Source)
2025-09-22 09:10:09.152 | 	at java.base/sun.nio.ch.NioSocketImpl.connect(Unknown Source)
2025-09-22 09:10:09.152 | 	at java.base/java.net.Socket.connect(Unknown Source)
2025-09-22 09:10:09.152 | 	at java.base/sun.net.NetworkClient.doConnect(Unknown Source)
2025-09-22 09:10:09.152 | 	at java.base/sun.net.www.http.HttpClient.openServer(Unknown Source)
2025-09-22 09:10:09.152 | 	at java.base/sun.net.www.http.HttpClient.openServer(Unknown Source)
2025-09-22 09:10:09.152 | 	at java.base/sun.net.www.http.HttpClient.<init>(Unknown Source)
2025-09-22 09:10:09.152 | 	at java.base/sun.net.www.http.HttpClient.New(Unknown Source)
2025-09-22 09:10:09.152 | 	at java.base/sun.net.www.http.HttpClient.New(Unknown Source)
2025-09-22 09:10:09.152 | 	at java.base/sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(Unknown Source)
2025-09-22 09:10:09.152 | 	at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect0(Unknown Source)
2025-09-22 09:10:09.152 | 	at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect(Unknown Source)
2025-09-22 09:10:09.152 | 	at java.base/sun.net.www.protocol.http.HttpURLConnection.connect(Unknown Source)
2025-09-22 09:10:09.152 | 	at java.base/sun.net.www.protocol.http.HttpURLConnection.getOutputStream0(Unknown Source)
2025-09-22 09:10:09.152 | 	at java.base/sun.net.www.protocol.http.HttpURLConnection.getOutputStream(Unknown Source)
2025-09-22 09:10:09.152 | 	at io.camunda.client.impl.oauth.OAuthCredentialsProvider.fetchCredentials(OAuthCredentialsProvider.java:214)
2025-09-22 09:10:09.152 | 	at io.camunda.client.impl.oauth.OAuthCredentialsProvider.lambda$shouldRetryRequest$0(OAuthCredentialsProvider.java:159)
2025-09-22 09:10:09.152 | 	at io.camunda.client.impl.oauth.OAuthCredentialsCache.withCache(OAuthCredentialsCache.java:134)
2025-09-22 09:10:09.152 | 	at io.camunda.client.impl.oauth.OAuthCredentialsProvider.shouldRetryRequest(OAuthCredentialsProvider.java:156)
2025-09-22 09:10:09.152 | 	at io.camunda.client.impl.RetriableStreamingFutureImpl.onError(RetriableStreamingFutureImpl.java:52)
2025-09-22 09:10:09.152 | 	at io.grpc.stub.ClientCalls$StreamObserverToCallListenerAdapter.onClose(ClientCalls.java:564)
2025-09-22 09:10:09.152 | 	at io.grpc.internal.ClientCallImpl.closeObserver(ClientCallImpl.java:565)
2025-09-22 09:10:09.152 | 	at io.grpc.internal.ClientCallImpl.access$100(ClientCallImpl.java:72)
2025-09-22 09:10:09.152 | 	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl$1StreamClosed.runInternal(ClientCallImpl.java:733)
2025-09-22 09:10:09.152 | 	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl$1StreamClosed.runInContext(ClientCallImpl.java:714)
2025-09-22 09:10:09.152 | 	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
2025-09-22 09:10:09.152 | 	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:133)
2025-09-22 09:10:09.152 | 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
2025-09-22 09:10:09.152 | 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
2025-09-22 09:10:09.152 | 	at java.base/java.lang.Thread.run(Unknown Source)
2025-09-22 09:10:09.152 |
2025-09-22 09:10:09.152 | 2025-09-22T07:10:09.152Z  WARN 1 --- [ult-executor-60] io.camunda.client.job.poller             : Failed to activate jobs for worker Azure Blob Storage and job type io.camunda:azure-blobstorage:1
2025-09-22 09:10:09.152 |
2025-09-22 09:10:09.152 | io.grpc.StatusRuntimeException: UNAUTHENTICATED: Expected authentication information to start with 'Basic '
2025-09-22 09:10:09.152 | 	at io.grpc.Status.asRuntimeException(Status.java:532)
2025-09-22 09:10:09.152 | 	at io.grpc.stub.ClientCalls$StreamObserverToCallListenerAdapter.onClose(ClientCalls.java:564)
2025-09-22 09:10:09.152 | 	at io.grpc.internal.ClientCallImpl.closeObserver(ClientCallImpl.java:565)
2025-09-22 09:10:09.152 | 	at io.grpc.internal.ClientCallImpl.access$100(ClientCallImpl.java:72)
2025-09-22 09:10:09.152 | 	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl$1StreamClosed.runInternal(ClientCallImpl.java:733)
2025-09-22 09:10:09.152 | 	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl$1StreamClosed.runInContext(ClientCallImpl.java:714)
2025-09-22 09:10:09.152 | 	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
2025-09-22 09:10:09.152 | 	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:133)
2025-09-22 09:10:09.152 | 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
2025-09-22 09:10:09.152 | 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
2025-09-22 09:10:09.152 | 	at java.base/java.lang.Thread.run(Unknown Source)
2025-09-22 09:10:09.152 | 
```

In my application:
 ```
Caused by: io.camunda.client.api.command.ProblemException: Failed with code 401: 'Unauthorized'. Details: 'class ProblemDetail {
    type: about:blank
    title: Unauthorized
    status: 401
    detail: An Authentication object was not found in the SecurityContext
    instance: /v2/process-definitions/search
}'
	at io.camunda.client.impl.http.ApiCallback.handleErrorResponse(ApiCallback.java:125) ~[camunda-client-java-8.8.0-alpha8.jar:8.8.0-alpha8]
	at io.camunda.client.impl.http.ApiCallback.completed(ApiCallback.java:64) ~[camunda-client-java-8.8.0-alpha8.jar:8.8.0-alpha8]
	at io.camunda.client.impl.http.ApiCallback.completed(ApiCallback.java:32) ~[camunda-client-java-8.8.0-alpha8.jar:8.8.0-alpha8]
	at org.apache.hc.core5.concurrent.BasicFuture.completed(BasicFuture.java:148) ~[httpcore5-5.3.2.jar:5.3.2]
	at org.apache.hc.core5.concurrent.ComplexFuture.completed(ComplexFuture.java:72) ~[httpcore5-5.3.2.jar:5.3.2]
	at org.apache.hc.client5.http.impl.async.InternalAbstractHttpAsyncClient$2$1.completed(InternalAbstractHttpAsyncClient.java:310) ~[httpclient5-5.4.1.jar:5.4.1]
	at org.apache.hc.core5.http.nio.support.AbstractAsyncResponseConsumer$1.completed(AbstractAsyncResponseConsumer.java:101) ~[httpcore5-5.3.2.jar:5.3.2]
	at org.apache.hc.core5.http.nio.entity.AbstractBinAsyncEntityConsumer.completed(AbstractBinAsyncEntityConsumer.java:87) ~[httpcore5-5.3.2.jar:5.3.2]
	at org.apache.hc.core5.http.nio.entity.AbstractBinDataConsumer.streamEnd(AbstractBinDataConsumer.java:83) ~[httpcore5-5.3.2.jar:5.3.2]
	at org.apache.hc.core5.http.nio.support.AbstractAsyncResponseConsumer.streamEnd(AbstractAsyncResponseConsumer.java:142) ~[httpcore5-5.3.2.jar:5.3.2]
	at org.apache.hc.client5.http.impl.async.HttpAsyncMainClientExec$1.streamEnd(HttpAsyncMainClientExec.java:284) ~[httpclient5-5.4.1.jar:5.4.1]
	at org.apache.hc.core5.http.impl.nio.ClientHttp1StreamHandler.dataEnd(ClientHttp1StreamHandler.java:276) ~[httpcore5-5.3.2.jar:5.3.2]
	at org.apache.hc.core5.http.impl.nio.ClientHttp1StreamDuplexer.dataEnd(ClientHttp1StreamDuplexer.java:366) ~[httpcore5-5.3.2.jar:5.3.2]
	at org.apache.hc.core5.http.impl.nio.AbstractHttp1StreamDuplexer.onInput(AbstractHttp1StreamDuplexer.java:338) ~[httpcore5-5.3.2.jar:5.3.2]
	at org.apache.hc.core5.http.impl.nio.AbstractHttp1IOEventHandler.inputReady(AbstractHttp1IOEventHandler.java:64) ~[httpcore5-5.3.2.jar:5.3.2]
	at org.apache.hc.core5.http.impl.nio.ClientHttp1IOEventHandler.inputReady(ClientHttp1IOEventHandler.java:41) ~[httpcore5-5.3.2.jar:5.3.2]
	at org.apache.hc.core5.reactor.InternalDataChannel.onIOEvent(InternalDataChannel.java:143) ~[httpcore5-5.3.2.jar:5.3.2]
	at org.apache.hc.core5.reactor.InternalChannel.handleIOEvent(InternalChannel.java:51) ~[httpcore5-5.3.2.jar:5.3.2]
	at org.apache.hc.core5.reactor.SingleCoreIOReactor.processEvents(SingleCoreIOReactor.java:176) ~[httpcore5-5.3.2.jar:5.3.2]
	at org.apache.hc.core5.reactor.SingleCoreIOReactor.doExecute(SingleCoreIOReactor.java:125) ~[httpcore5-5.3.2.jar:5.3.2]
	at org.apache.hc.core5.reactor.AbstractSingleCoreIOReactor.execute(AbstractSingleCoreIOReactor.java:92) ~[httpcore5-5.3.2.jar:5.3.2]
	at org.apache.hc.core5.reactor.IOReactorWorker.run(IOReactorWorker.java:44) ~[httpcore5-5.3.2.jar:5.3.2]
	at java.base/java.lang.Thread.run(Thread.java:1583) ~[na:na]
```
with this configuration
```yaml
camunda:
  client:
    enabled: true
    rest-address: http://localhost:8088
    grpc-address: http://localhost:26500
    mode: selfmanaged
    auth:
      token-url: http://localhost:18080/auth/realms/camunda-platform/protocol/openid-connect/token
      client-id: zeebe
      client-secret: zecret
      connect-timeout: PT10S
      read-timeout: PT30S
```

--- a/docker-compose/versions/camunda-8.10/README.md
+++ b/docker-compose/versions/camunda-8.10/README.md
@@ -45,6 +45,8 @@ The Camunda Docker image automatically loads any `.jar` dropped into `/driver-li
 
 ## Enabling multi-tenancy
 
+### Lightweight configuration
+
 The lightweight `docker-compose.yaml` runs with basic authentication and an unprotected API. To enable multi-tenancy, protect the API and switch on the tenancy checks by creating a `docker-compose.override.yaml` next to the compose file:
 
 ```yaml
@@ -72,3 +74,24 @@ curl -u demo:demo -X PUT http://localhost:8080/v2/tenants/tenant-a/users/demo
 ```
 
 API clients must authenticate with basic auth once the API is protected (`camunda.client.auth.method=basic` plus username and password in the Camunda client SDKs).
+
+### Full configuration
+
+The full `docker-compose-full.yaml` already protects the API through Keycloak, so only the tenancy checks need to be switched on. Add the following to `.env`:
+
+```bash
+CAMUNDA_SECURITY_MULTITENANCY_CHECKSENABLED=true
+CAMUNDA_SECURITY_MULTITENANCY_APIENABLED=true
+```
+
+Then start the stack with `docker compose -f docker-compose-full.yaml up -d` and manage tenants through the Orchestration Cluster API with an OAuth token (or the Identity UI at `http://localhost:8080/identity`):
+
+```bash
+TOKEN=$(curl -s -X POST 'http://localhost:18080/auth/realms/camunda-platform/protocol/openid-connect/token' \
+  -d 'grant_type=client_credentials' -d 'client_id=orchestration' -d 'client_secret=secret' | jq -r .access_token)
+# create a tenant
+curl -X POST http://localhost:8080/v2/tenants -H "Authorization: Bearer $TOKEN" \
+  -H 'Content-Type: application/json' -d '{"tenantId": "tenant-a", "name": "Tenant A"}'
+# assign the demo user to it
+curl -X PUT http://localhost:8080/v2/tenants/tenant-a/users/demo -H "Authorization: Bearer $TOKEN"
+```

--- a/docker-compose/versions/camunda-8.10/README.md
+++ b/docker-compose/versions/camunda-8.10/README.md
@@ -42,3 +42,33 @@ docker compose up -d
 ### JDBC drivers
 
 The Camunda Docker image automatically loads any `.jar` dropped into `/driver-lib`. A writable `driver-lib/` folder is included next to the compose file so you can copy the vendor JDBC driver there before starting (e.g., `driver-lib/mysql-connector-j-9.0.0.jar`). This is required for MySQL/MariaDB, SQL Server, and Oracle. PostgreSQL and H2 drivers are already bundled.
+
+## Enabling multi-tenancy
+
+The lightweight `docker-compose.yaml` runs with basic authentication and an unprotected API. To enable multi-tenancy, protect the API and switch on the tenancy checks by creating a `docker-compose.override.yaml` next to the compose file:
+
+```yaml
+services:
+  orchestration:
+    environment:
+      - CAMUNDA_SECURITY_AUTHENTICATION_UNPROTECTEDAPI=false
+      - CAMUNDA_SECURITY_MULTITENANCY_CHECKSENABLED=true
+      - CAMUNDA_SECURITY_MULTITENANCY_APIENABLED=true
+  connectors:
+    environment:
+      - CAMUNDA_CLIENT_AUTH_METHOD=basic
+      - CAMUNDA_CLIENT_AUTH_USERNAME=demo
+      - CAMUNDA_CLIENT_AUTH_PASSWORD=demo
+```
+
+Then start the stack with `docker compose up -d` and manage tenants through the Orchestration Cluster API (or the Identity UI at `http://localhost:8080/identity`):
+
+```bash
+# create a tenant
+curl -u demo:demo -X POST http://localhost:8080/v2/tenants \
+  -H 'Content-Type: application/json' -d '{"tenantId": "tenant-a", "name": "Tenant A"}'
+# assign the demo user to it
+curl -u demo:demo -X PUT http://localhost:8080/v2/tenants/tenant-a/users/demo
+```
+
+API clients must authenticate with basic auth once the API is protected (`camunda.client.auth.method=basic` plus username and password in the Camunda client SDKs).

--- a/docker-compose/versions/camunda-8.10/README.md
+++ b/docker-compose/versions/camunda-8.10/README.md
@@ -63,7 +63,7 @@ services:
       - CAMUNDA_CLIENT_AUTH_PASSWORD=demo
 ```
 
-Then start the stack with `docker compose up -d` and manage tenants through the Orchestration Cluster API (or the Identity UI at `http://localhost:8080/identity`):
+Then start the stack with `docker compose up -d` and manage tenants through the Orchestration Cluster API (or the Orchestration Cluster Admin UI at `http://localhost:8080/admin`):
 
 ```bash
 # create a tenant
@@ -84,7 +84,7 @@ CAMUNDA_SECURITY_MULTITENANCY_CHECKSENABLED=true
 CAMUNDA_SECURITY_MULTITENANCY_APIENABLED=true
 ```
 
-Then start the stack with `docker compose -f docker-compose-full.yaml up -d` and manage tenants through the Orchestration Cluster API with an OAuth token (or the Identity UI at `http://localhost:8080/identity`):
+Then start the stack with `docker compose -f docker-compose-full.yaml up -d` and manage tenants through the Orchestration Cluster API with an OAuth token (or the Orchestration Cluster Admin UI at `http://localhost:8080/admin`):
 
 ```bash
 TOKEN=$(curl -s -X POST 'http://localhost:18080/auth/realms/camunda-platform/protocol/openid-connect/token' \

--- a/docker-compose/versions/camunda-8.7/README.md
+++ b/docker-compose/versions/camunda-8.7/README.md
@@ -3,3 +3,22 @@
 ## Usage
 
 For end user usage, please check the offical documentation of [Camunda 8 Self-Managed Docker Compose](https://docs.camunda.io/docs/self-managed/setup/deploy/local/docker-compose/).
+
+## Enabling multi-tenancy
+
+Multi-tenancy requires Zeebe to authenticate through Identity. Set the following in `.env`:
+
+```bash
+ZEEBE_AUTHENTICATION_MODE=identity
+MULTI_TENANCY_ENABLED=true
+```
+
+Then start the stack with `docker compose up -d` and manage tenants in the Identity UI at `http://localhost:8084` (log in as `demo`/`demo`), or through the Identity API:
+
+```bash
+# create a tenant (token from the camunda-identity client, e.g. via password grant for the demo user)
+curl -X POST http://localhost:8084/api/tenants -H "Authorization: Bearer $TOKEN" \
+  -H 'Content-Type: application/json' -d '{"tenantId": "tenant-a", "name": "Tenant A"}'
+```
+
+API clients must use OAuth client credentials (e.g. the `zeebe` client from `.env`) once authentication is enabled.

--- a/docker-compose/versions/camunda-8.8/README.md
+++ b/docker-compose/versions/camunda-8.8/README.md
@@ -3,3 +3,33 @@
 ## Usage
 
 For end user usage, please check the offical documentation of [Camunda 8 Self-Managed Docker Compose](https://docs.camunda.io/docs/next/self-managed/quickstart/developer-quickstart/docker-compose/).
+
+## Enabling multi-tenancy
+
+The lightweight `docker-compose.yaml` runs with basic authentication and an unprotected API. To enable multi-tenancy, protect the API and switch on the tenancy checks by creating a `docker-compose.override.yaml` next to the compose file:
+
+```yaml
+services:
+  orchestration:
+    environment:
+      - CAMUNDA_SECURITY_AUTHENTICATION_UNPROTECTEDAPI=false
+      - CAMUNDA_SECURITY_MULTITENANCY_CHECKSENABLED=true
+      - CAMUNDA_SECURITY_MULTITENANCY_APIENABLED=true
+  connectors:
+    environment:
+      - CAMUNDA_CLIENT_AUTH_METHOD=basic
+      - CAMUNDA_CLIENT_AUTH_USERNAME=demo
+      - CAMUNDA_CLIENT_AUTH_PASSWORD=demo
+```
+
+Then start the stack with `docker compose up -d` and manage tenants through the Orchestration Cluster API (or the Identity UI at `http://localhost:8088/identity`):
+
+```bash
+# create a tenant
+curl -u demo:demo -X POST http://localhost:8088/v2/tenants \
+  -H 'Content-Type: application/json' -d '{"tenantId": "tenant-a", "name": "Tenant A"}'
+# assign the demo user to it
+curl -u demo:demo -X PUT http://localhost:8088/v2/tenants/tenant-a/users/demo
+```
+
+API clients must authenticate with basic auth once the API is protected (`camunda.client.auth.method=basic` plus username and password in the Camunda client SDKs).

--- a/docker-compose/versions/camunda-8.8/README.md
+++ b/docker-compose/versions/camunda-8.8/README.md
@@ -6,6 +6,8 @@ For end user usage, please check the offical documentation of [Camunda 8 Self-Ma
 
 ## Enabling multi-tenancy
 
+### Lightweight configuration
+
 The lightweight `docker-compose.yaml` runs with basic authentication and an unprotected API. To enable multi-tenancy, protect the API and switch on the tenancy checks by creating a `docker-compose.override.yaml` next to the compose file:
 
 ```yaml
@@ -33,3 +35,24 @@ curl -u demo:demo -X PUT http://localhost:8088/v2/tenants/tenant-a/users/demo
 ```
 
 API clients must authenticate with basic auth once the API is protected (`camunda.client.auth.method=basic` plus username and password in the Camunda client SDKs).
+
+### Full configuration
+
+The full `docker-compose-full.yaml` already protects the API through Keycloak, so only the tenancy checks need to be switched on. Add the following to `.env`:
+
+```bash
+CAMUNDA_SECURITY_MULTITENANCY_CHECKSENABLED=true
+CAMUNDA_SECURITY_MULTITENANCY_APIENABLED=true
+```
+
+Then start the stack with `docker compose -f docker-compose-full.yaml up -d` and manage tenants through the Orchestration Cluster API with an OAuth token (or the Identity UI at `http://localhost:8088/identity`):
+
+```bash
+TOKEN=$(curl -s -X POST 'http://localhost:18080/auth/realms/camunda-platform/protocol/openid-connect/token' \
+  -d 'grant_type=client_credentials' -d 'client_id=orchestration' -d 'client_secret=secret' | jq -r .access_token)
+# create a tenant
+curl -X POST http://localhost:8088/v2/tenants -H "Authorization: Bearer $TOKEN" \
+  -H 'Content-Type: application/json' -d '{"tenantId": "tenant-a", "name": "Tenant A"}'
+# assign the demo user to it
+curl -X PUT http://localhost:8088/v2/tenants/tenant-a/users/demo -H "Authorization: Bearer $TOKEN"
+```

--- a/docker-compose/versions/camunda-8.9/README.md
+++ b/docker-compose/versions/camunda-8.9/README.md
@@ -3,3 +3,33 @@
 ## Usage
 
 For end user usage, please check the offical documentation of [Camunda 8 Self-Managed Docker Compose](https://docs.camunda.io/docs/next/self-managed/quickstart/developer-quickstart/docker-compose/).
+
+## Enabling multi-tenancy
+
+The lightweight `docker-compose.yaml` runs with basic authentication and an unprotected API. To enable multi-tenancy, protect the API and switch on the tenancy checks by creating a `docker-compose.override.yaml` next to the compose file:
+
+```yaml
+services:
+  orchestration:
+    environment:
+      - CAMUNDA_SECURITY_AUTHENTICATION_UNPROTECTEDAPI=false
+      - CAMUNDA_SECURITY_MULTITENANCY_CHECKSENABLED=true
+      - CAMUNDA_SECURITY_MULTITENANCY_APIENABLED=true
+  connectors:
+    environment:
+      - CAMUNDA_CLIENT_AUTH_METHOD=basic
+      - CAMUNDA_CLIENT_AUTH_USERNAME=demo
+      - CAMUNDA_CLIENT_AUTH_PASSWORD=demo
+```
+
+Then start the stack with `docker compose up -d` and manage tenants through the Orchestration Cluster API (or the Identity UI at `http://localhost:8080/identity`):
+
+```bash
+# create a tenant
+curl -u demo:demo -X POST http://localhost:8080/v2/tenants \
+  -H 'Content-Type: application/json' -d '{"tenantId": "tenant-a", "name": "Tenant A"}'
+# assign the demo user to it
+curl -u demo:demo -X PUT http://localhost:8080/v2/tenants/tenant-a/users/demo
+```
+
+API clients must authenticate with basic auth once the API is protected (`camunda.client.auth.method=basic` plus username and password in the Camunda client SDKs).

--- a/docker-compose/versions/camunda-8.9/README.md
+++ b/docker-compose/versions/camunda-8.9/README.md
@@ -24,7 +24,7 @@ services:
       - CAMUNDA_CLIENT_AUTH_PASSWORD=demo
 ```
 
-Then start the stack with `docker compose up -d` and manage tenants through the Orchestration Cluster API (or the Identity UI at `http://localhost:8080/identity`):
+Then start the stack with `docker compose up -d` and manage tenants through the Orchestration Cluster API (or the Orchestration Cluster Admin UI at `http://localhost:8080/admin`):
 
 ```bash
 # create a tenant
@@ -45,7 +45,7 @@ CAMUNDA_SECURITY_MULTITENANCY_CHECKSENABLED=true
 CAMUNDA_SECURITY_MULTITENANCY_APIENABLED=true
 ```
 
-Then start the stack with `docker compose -f docker-compose-full.yaml up -d` and manage tenants through the Orchestration Cluster API with an OAuth token (or the Identity UI at `http://localhost:8080/identity`):
+Then start the stack with `docker compose -f docker-compose-full.yaml up -d` and manage tenants through the Orchestration Cluster API with an OAuth token (or the Orchestration Cluster Admin UI at `http://localhost:8080/admin`):
 
 ```bash
 TOKEN=$(curl -s -X POST 'http://localhost:18080/auth/realms/camunda-platform/protocol/openid-connect/token' \

--- a/docker-compose/versions/camunda-8.9/README.md
+++ b/docker-compose/versions/camunda-8.9/README.md
@@ -6,6 +6,8 @@ For end user usage, please check the offical documentation of [Camunda 8 Self-Ma
 
 ## Enabling multi-tenancy
 
+### Lightweight configuration
+
 The lightweight `docker-compose.yaml` runs with basic authentication and an unprotected API. To enable multi-tenancy, protect the API and switch on the tenancy checks by creating a `docker-compose.override.yaml` next to the compose file:
 
 ```yaml
@@ -33,3 +35,24 @@ curl -u demo:demo -X PUT http://localhost:8080/v2/tenants/tenant-a/users/demo
 ```
 
 API clients must authenticate with basic auth once the API is protected (`camunda.client.auth.method=basic` plus username and password in the Camunda client SDKs).
+
+### Full configuration
+
+The full `docker-compose-full.yaml` already protects the API through Keycloak, so only the tenancy checks need to be switched on. Add the following to `.env`:
+
+```bash
+CAMUNDA_SECURITY_MULTITENANCY_CHECKSENABLED=true
+CAMUNDA_SECURITY_MULTITENANCY_APIENABLED=true
+```
+
+Then start the stack with `docker compose -f docker-compose-full.yaml up -d` and manage tenants through the Orchestration Cluster API with an OAuth token (or the Identity UI at `http://localhost:8080/identity`):
+
+```bash
+TOKEN=$(curl -s -X POST 'http://localhost:18080/auth/realms/camunda-platform/protocol/openid-connect/token' \
+  -d 'grant_type=client_credentials' -d 'client_id=orchestration' -d 'client_secret=secret' | jq -r .access_token)
+# create a tenant
+curl -X POST http://localhost:8080/v2/tenants -H "Authorization: Bearer $TOKEN" \
+  -H 'Content-Type: application/json' -d '{"tenantId": "tenant-a", "name": "Tenant A"}'
+# assign the demo user to it
+curl -X PUT http://localhost:8080/v2/tenants/tenant-a/users/demo -H "Authorization: Bearer $TOKEN"
+```

